README.md

```diff
@@ -17,6 +17,6 @@ https://www.markdownguide.org/cheat-sheet/
 
 
 ## Topics
-* git ignore
+* Git ignore
```

Comment:
shouldn't the I be caps also?

Reply:
fixed

Reply:
https://media.giphy.com/media/RddAJiGxTPQFa/giphy.gif

Reply:
![giphy](https://user-images.githubusercontent.com/1104889/100549290-afdcb800-3237-11eb-9b9f-ea7b067b36d4.gif)